shoop/xtheme/parsing.py

```diff
@@ -171,6 +173,10 @@ class PlaceholderExtension(_PlaceholderManagingExtension):
     * The body of this block is actually discarded; only the inner `column`, `row` and `plugin`
       directives have any meaning.  (A parser-time `Layout` object is created and populated during parsing
       of this block.)
+    * An optional `global` parameter can be used to specify that the configuration for this placeholder
+      should be "global" across different views (although currently that only applies to placeholders within
+      the same template, i.e, if you defined the same global placeholder in different templates they will
+      not share configuration, but an included or base template that is rendered by different views will.
```

Comment:
Docstrings should be wrapped to narrower width (about column 72 or so, max 79) to make them easier to read and review. (Yeah, I know, the text above your added part is not wrapped so. You may change it too in a separate commit or leave it as is.)